### ISSUE TITLE
Unittests take 8 seconds to run.

### ISSUE THREAD

**Issue by dhermes** [google/apitools#6]
I'm going to use the `nose.tools.timed(limit)` decorator as in
https://github.com/google/oauth2client/issues/85
https://github.com/google/oauth2client/pull/89
to determine where the slow test is and why.

**Comment by dhermes** [comment]
Not a surprise, after [decorating][1] with a 0.1 second timer, the only method to exceed is:

```
FAIL: testGeneration (apitools.gen.client_generation_test.ClientGenerationTest)
```

As it is, #5 makes this test only run under Python 2.7, so I'd suggest moving it to a standalone, just like what is done for the `gcloud-python` regression tests (which are actually system tests).


[1]: https://gist.github.com/dhermes/a6616d9d14098e57e03f

**Comment by craigcitro** [comment]
yeah, i'm down with moving out this one test. (i'll do it if you don't beat me to it.)

**Comment by dhermes** [comment]
I won't beat you to it
